python/ray/scripts/scripts.py

```diff
@@ -572,12 +583,29 @@ def start(node_ip_address, address, port, redis_password, redis_shard_ports,
             # not provided.
             num_redis_shards = len(redis_shard_ports)
 
-        if address is not None:
+        if address is not None or external_kv_storage_config is not None:
             cli_logger.print(
                 "Will use value of `{}` as remote Redis server address(es). "
                 "If the primary one is not reachable, we starts new one(s) "
-                "with `{}` in local.", cf.bold("--address"), cf.bold("--port"))
-            external_addresses = address.split(",")
+                "with `{}` in local.", cf.bold("--external-kv-storage-config"),
+                cf.bold("--port"))
+            external_addresses = []
+            if external_kv_storage_config is not None:
+                re_obj = re.match(r"(redis):(.*)", external_kv_storage_config,
```

Comment:
I think prefix matching `redis:` should be sufficient here. But it is minor and we can always come back to update this.

Reply:
OK. Let's leave the code as it is in this PR.